Sources/Regex/Parse/Token.swift

```diff
@@ -79,13 +89,19 @@ extension Token.MetaCharacter: CustomStringConvertible {
 extension Token.SetOperator: CustomStringConvertible {
   var description: String { rawValue }
 }
+extension Token.SpecialCharacterEscape: CustomStringConvertible {
+  var description: String { rawValue }
+}
 extension Token: CustomStringConvertible {
   var description: String {
     switch self {
     case .meta(let meta): return meta.description
     case .setOperator(let op): return op.description
     case .character(let c, _): return c.halfWidthCornerQuoted
     case .unicodeScalar(let u): return "U\(u.halfWidthCornerQuoted)"
+    case .builtinCharClass(let cc): return cc.description
+    case .anchor(let anchor): return anchor.description
+    case .specialCharEscape(let special): return special.description
```

Comment:
What is "special" about this?

Reply:
It's just what I decided to call it, I don't suppose you have any suggestions on an alternative name?